offscreen/manual.md

```diff
@@ -0,0 +1,140 @@
+having a look at the [ordering example](../ordering/manual.md) to understand how to structure a scene in different
+parts which are rendered at different
+times - this technique is a requirement for offscreen rendering.
+
+Let's have a look at the contents of the project.
+
+## Scene graph and resources
+
+After opening the project, we notice a few more things compared to the [monkey example](../monkey/manual.md).
+
+To keep this example simple, we render a single monkey head without a script.
+In the Scene View, there are two cameras - one for rendering the monkey, one for rendering the quad:
+
+![](./docs/cameras.png)
+
+Furthermore, there are a bunch of new objects in the Resource View:
+
+![](./docs/resources.png)
+
+These new objects define how the offscreen rendering should work, i.e. which objects are
+rendered to which buffer at which time.
+
+Let's start with the RenderLayers. We need two such layers - one for the monkey, one for the quad.
+Each of them uses a different tag to filter their content - `monkey` and `quad` respectively.
+
+Here is how the `RenderLayer` of the monkey looks like this (it uses a `monkey` tag to collect the monkey mesh):
+
+![](./docs/renderlayer.png)
+
+Next, we have two `RenderPasses`, one for each `RenderLayer`. The `RenderPass` which renders the monkey
+uses an offscreen `RenderTarget`, while the one of the quad renders to the framebuffer:
+
+![](./docs/monkeypass.png)
+
+![](./docs/quadpass.png)
+
+Note that the `RenderPass` of the monkey also clears its `RenderTarget`, and uses a clear color.
+In Ramses, the framebuffer
+settings are not managed by the scene, but is rather a global setting of the display where the scene is being shown.
+`RenderTargets` belong to the scene, and therefore must be explicitly cleared.
+
+Let's have a look at the `RenderTarget`:
+
+![](./docs/rendertarget.png)
+
+The `RenderTarget` (in OpenGL terms - a [framebuffer object](https://www.khronos.org/opengl/wiki/Framebuffer_Object), or FBO)
+is a collection of one or multiple buffers for rendering.
+Those can be color buffers, and optionally a depth and/or stencil buffer. These buffers must all have the same size, and must
+fulfill other criteria too - e.g. you can have at most one depth buffer, and they can't exceed a given size. The Ramses Composer
+will report errors in the configuration of the `RenderTarget` if such are found. Note that the FBO configuration is a hardware
+detail - not all formats work on all hardware devices. Some devices have a limit on e.g. how many floating point textures can
+be created, and not all devices support 16 or 32 bit float channels. Be careful which formats you really need and cross-check
+with your target hardware!
+
+In this example, we have two such buffers - one for color, and one for depth information. Note that they have
+a size which is different than the framebuffer, and have a format (RGBA8 and Depth24 respectively):
```

Comment:
what is the framebuffer size in this example, or how can I tell that it is different?

Reply:
yes, better now